README.md

```diff
@@ -27,6 +29,17 @@ Metrics will now be accessible at http://localhost:8080/metrics
 
 To bind the java agent to a specific IP change the port number to `host:port`.
 
+__Note:__ Within the config.yaml...
+
+```
+hostPort: localhost:5555
+username: 
+password:
+```
+... is typically not required.
+
+### As an HTTP Server
```

Comment:
As I'm thinking about it, _HTTP Server_ seems to be a bad name, as it implies the agent does not have an HTTP server (see #671).

It's hard to change that consistently, as the `jmx_prometheus_httpserver` module literally has `httpserver` in its name. However, maybe we should call this section "Standalone mode" as opposed to "agent mode"?

Reply:
`Standalone mode` sounds great.